src/language/worker.ts

```diff
@@ -219,6 +219,6 @@ const handleMessageFromParentPort = async(message: string) => {
 parentPort?.once(
   'message',
   (message: string): void => {
-    handleMessageFromParentPort(message);
+    void handleMessageFromParentPort(message);
```

Comment:
what happens if this promise rejects? .. also you can make the callback async

Reply:
So as we discussed in slack, let's move this additional refactoring out of the scope of this PR. The current PR introduces a tool that helps you identify where is an intentional void function call and where is accidentally missed await. We can check all void cases later together with @Anemy since we share the code of the extension and update these cases. Please create a ticket for this! I also will create one to finish replacing promises with async/await syntax.